v1/core/support-check/1.0/index.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * REST API Endpoint used to check the recommended ClassicPress PHP version.
+ * This is similar to http://api.wordpress.org/core/serve-happy/1.0/ 
+ *
+ * Response should be an array with:
+ *  'recommended_version' - string - The PHP version recommended by ClassicPress.
+ *  'is_supported' - boolean - Whether the PHP version is actively supported.
+ *  'is_secure' - boolean - Whether the PHP version receives security updates.
+ *  'is_acceptable' - boolean - Whether the PHP version is still acceptable for ClassicPress.
+ */
+$json = [
+    'recommended_version' => '5.6',
```

Comment:
Why is recommended version 5.6? Shouldn't this be 7.4?

Reply:
I need re-evaluate this. I have found some new information.

Reply:
I have set 7.4 as recommended but I think we might be bumping that up soon